cms/articles/migrations/0005_statisticalarticlepage_restucture_headline_figures.py

```diff
@@ -0,0 +1,74 @@
+# Generated by Django 5.1.8 on 2025-05-06 10:17
```

Comment:
note that this will clash with https://github.com/ONSdigital/dis-wagtail/pull/165/files#diff-917e835e7853827ccf5fac6e7143b52763469aad7c3b98e9077e766fab62aba8R1

@AdamHawtin there was one outstanding bit to fix on the PR before we can merge it. Given yours was open for a bit can you coordinate with @MaciekBaron on the order of operations?